src/EventStore.Core.Tests/Helpers/MiniClusterNode.cs

```diff
@@ -142,9 +146,16 @@ public class MiniClusterNode<TLogFormat, TStreamId> {
 				Projection = new() {
 					RunProjections = ProjectionType.None
 				},
-				Subsystems = subsystems ?? Array.Empty<ISubsystem>()
+				Subsystems = subsystems
 			};
 
+			var inMemConf = new ConfigurationBuilder()
+				.AddInMemoryCollection(new KeyValuePair<string, string>[] {
+					new("EventStore:Plugins:Tcp:ExtPort", externalTcp.Port.ToString()),
+					new("EventStore:Plugins:Tcp:ExtHeartbeatInterval", "10000"),
+					new("EventStore:Plugins:Tcp:ExtHeartbeatTimeout", "10000"),
+					new("EventStore:Plugins:Tcp:Insecure", options.Application.Insecure.ToString()),
```

Comment:
Probably we shouldn't have an extra setting for Tcp Insecure, and instead just have the normal server one (otherwise in real life people will have to coordinate the configuration of their tcp plugin with the server configuration)

Reply:
It's only used for testing. The TCP plugin uses the database settings to set this up.

Reply:
ah i see, perhaps if we make it `EventStore:Plugins:TestTcp:...`